### PR TITLE
Audio Player component

### PR DESCRIPTION
**Pull request by peterpme**
image

image

### CODE REVIEW

File: src/components/AudioPlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  layout: {},
+  props: {
+    source: {
+      group: GROUPS.data,
+      label: "URL",
+      description: "The url of the sound",
+      editable: true,
+      required: true,
+      defaultValue:
+        "https://hwcdn.libsyn.com/p/9/e/2/9e224ffb4f3625c8/Nerdist_644_-_Ted_Melfi.mp3?c_id=8473543&cs_id=8473543&destination_id=18174&expiration=1607628092&hwt=f8327d4f551ec81f902e666647dc76f5",
```
Comment (peterpme) [inline src/components/AudioPlayer.tsx:173]: could probably remove this

File: src/components/AspectRatio.web.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const AspectRatio: React.FC<Props> = (props) => {
     <View
       {...props}
       style={style}
-      onLayout={({ nativeEvent: { layout } }) => setLayout(layout)}
+      onLayout={({ nativeEvent: { layout: l } }) => setLayout(l)}
```
[the remark this answers is not in the document]
Comment (peterpme) [reply src/components/AspectRatio.web.tsx:33]: fixes an error that prevented me from publishing, commitlint is strict

File: src/components/AudioPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
+// @ts-nocheck
```
[the remark this answers is not in the document]
Comment (peterpme) [reply src/components/AudioPlayer.tsx:1]: not before nick's demo

[the remark this answers is not in the document]
Comment (peterpme) [reply src/components/AudioPlayer.tsx:1]: i would have to go into expo and figure out how to type shit like source and sound

File: src/components/AspectRatio.web.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (dhruska) [reply src/components/AspectRatio.web.tsx:33]: sounds good, I thought the `l` was a `1` lol

**Review by dhruska — APPROVED** [review]

File: src/components/AspectRatio.web.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (peterpme) [reply src/components/AspectRatio.web.tsx:33]: lol

**Pull request merged**